hana_recipe.yaml

```
@@ -0,0 +1,16 @@
+source:
+  type: datahub_sap_hana.ingestion.HanaSource
+  config:
+    username: admin
+    password: password123
+    database: db_name
+    host_port: localhost:39001
+    schema_pattern:
+      deny: 
+        - .*
```

I don't think this is a good default example, should it be deny for SYS?
```suggestion
        - .*SYS*
```

---

I removed .*SYS*  bec I thought I should remove the references to our sample db